oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1725,6 +1728,7 @@ void pldm::responder::oem_ibm_platform::Handler::processPowerOffHardGraceful()
             << "Setting one-time restore policy failed, Unable to set property PowerRestorePolicy. ERROR="
             << e.what() << "\n";
     }
+    processPowerCycleOffSoftGraceful();
```

Comment:
Still don't think this is right. It should be calling "processPowerOffSoftGraceful". We want to arm APR and then chassis power off the system. My state-manager code will call APR to power back on once UPS is fixed.